presto-main/src/main/java/com/facebook/presto/type/DecimalCasts.java

```diff
@@ -464,6 +463,7 @@ public static long floatToShortDecimal(long value, long precision, long scale, l
     @UsedByGeneratedCode
     public static Slice floatToLongDecimal(long value, long precision, long scale, long tenToScale)
     {
+        // TODO: optimize
```

Comment:
This comment is to general, nobody, but you (today), knows what can be optimized.
Can you please explain what and how can be optimized?

Or remove it.